_life/money.md

```diff
@@ -79,75 +71,74 @@ Do not expect good yields from non-investment bank accounts, even the savings on
 
 While you technically can cash checks, banks will charge large fees for doing so, as virtually no one in Switzerland uses checks any more.
 
+
 ## Internships in the US
 
-The most likely complication in your personal finances will be if you do an
-internship in the US. Luckily, this guide has you covered with [banking, taxes, and
-other advice](/life/money-americans#internships-in-the-us-for-non-citizens) for that
-circumstance.
+The most likely complication in your personal finances will be if you do an internship in the US.
+Luckily, this guide has you covered with [banking, taxes, and other advice](/life/money-americans#internships-in-the-us-for-non-citizens) for that circumstance.
+
 
 ## Retirement
 
 Switzerland has a system based on three pillars: shared mandatory, personal mandatory, and personal optional.
 
-### First pillar
-
-Mandatory retirement savings, taken out of the salary, as a global pool for everyone. Called "AVS". Equivalent to "social security".
-
-### Second pillar
-
-Mandatory retirement savings, taken out of the salary, for yourself. The provider is always [Publica](https://publica.ch/) for EPFL employees.
-Money is blocked until retirement or "permanently" leaving the country.
-
-### Third pillar
+The first pillar is mandatory retirement savings, taken out of the salary, as a global pool for everyone. Called "AVS", equivalent to "social security".
 
-Optional retirement savings for yourself. The money contributed to the Pillar 3a can be deducted from income for tax purposes.
-Capped to about CHF 7000 per year (the exact amount increases slightly every year).
-You can view this as an investment with a high return since its tax-deductible nature drastically lowers your taxes (see below).
-
-Money is blocked until retirement, "permanently" leaving the country, buying a house or starting to work as an independent.
-At withdrawal, a progressive tax rate is applied depending on the capital amount and your canton of residence. ([source](https://finpension.ch/en/capital-withdrawal-tax-compared/))
+The secpond pillar is also mandatory retirement savings, taken out of the salary, but for yourself.
+The provider is always [Publica](https://publica.ch/) for EPFL employees.
+Money is blocked until you retire, "permanently" leave the country, use it to buy a house, or use it to start a company.
```

Comment:
```suggestion
The second pillar is also mandatory retirement savings, taken out of the salary, but for yourself.
The provider is always [PUBLICA](https://publica.ch/) for EPFL employees.
Money is blocked until you retire, "permanently" leave the country, use it to buy a house, or use it to start a company.
```

worth clarifying what "permanently" means here perhaps

Reply:
Good idea. I think some form of "your permit isn't renewed" would be a good succinctness/precision tradeoff, but I'm not sure how to phrase this without sounding negative (you effectively get to influence this by getting a job in Switzerland, so it's not like you'll be kicked out)

Reply:
Don't you have to declare by yourself that you are "permanently" leaving the country?

First pillar [from ch.ch](https://www.ch.ch/en/retirement/oasi-pension-abroad/): “If Switzerland does not have a social security agreement with the country where you live that provides for your pension to be paid to you there, when you leave Switzerland you are only entitled to an interest-free refund of what you and your employer(s) have paid into the OASI fund. If you have to apply for this refund, you will find the form and the information you require on the website of the Swiss Compensation Office in Geneva.”

Second pillar [from PUBLICA](https://www.publica.ch/en/my-pension/life-events/moving-abroad): “Please tell us if you are planning to leave Switzerland for good, and read our page on information requirements.”

Reply:
Yeah... and given that the agreements change all the time, I guess I'll just add a vague sentence.